Sources/NIOHTTP2/HTTP2ChannelHandler+InlineStreamMultiplexer.swift

```diff
@@ -211,3 +211,39 @@ extension InlineStreamMultiplexer {
         self.commonStreamMultiplexer.setChannelContinuation(streamChannels)
     }
 }
+
+extension NIOHTTP2Handler {
+    /// A variant of `NIOHTTP2Handler.StreamMultiplexer` which creates a child channel for each HTTP/2 stream and
+    /// provides access to inbound HTTP/2 streams.
+    ///
+    /// In general in NIO applications it is helpful to consider each HTTP/2 stream as an
+    /// independent stream of HTTP/2 frames. This multiplexer achieves this by creating a
+    /// number of in-memory `HTTP2StreamChannel` objects, one for each stream. These operate
+    /// on ``HTTP2Frame/FramePayload`` objects as their base communication
+    /// atom, as opposed to the regular NIO `SelectableChannel` objects which use `ByteBuffer`
+    /// and `IOData`.
+    ///
+    /// Locally-initiated stream channel objects are initialized upon creation using the supplied `initializer` which returns a type
```

Comment:
This is hinting at the `openStream` method right? We should also talk about remote initiated stream here which use the `InboundStreamOutput` generic type (maybe we should rename that type?)

Reply:
We can also explicitly spell out how the users can open streams. (i.e. "You can open a stream by calling ....")